cas-server-support-duo/src/main/java/com/duosecurity/duoweb/Util.java

```diff
@@ -7,6 +7,15 @@
 import javax.crypto.spec.SecretKeySpec;
 
 public class Util {
+    /**
+     * TODO: 12/10/15   move to DuoWeb
```

Comment:
I would caution you against doing so; these components exist here because Duo sadly does not release their artifacts to Maven Central, which requires us to keep their code here locally and release/compile it locally. Which means we take what they have verbaitm without any changes because we want to continue doing so when they "release" new updates to the code, and that's just the easiest thing to. 

It's up to you; I am suggesting what might work easier in the long term. Once Duo understands the benefits of a public repository!, all of this will be removed by a dependency to their artifact.